feature_tests/kotlin/somelib/src/main/kotlin/dev/diplomattest/somelib/CallbackWrapper.kt

```diff
@@ -0,0 +1,281 @@
+package dev.diplomattest.somelib
+
+import com.sun.jna.Callback
+import com.sun.jna.Library
+import com.sun.jna.Native
+import com.sun.jna.Pointer
+import com.sun.jna.Structure
+
+internal interface CallbackWrapperLib: Library {
+    fun CallbackWrapper_test_multi_arg_callback(f: DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomatCallback_f_Native, x: Int): Int
+    fun CallbackWrapper_test_no_args(h: DiplomatCallback_CallbackWrapper_test_no_args_diplomatCallback_h_Native): Int
+    fun CallbackWrapper_test_cb_with_struct(f: DiplomatCallback_CallbackWrapper_test_cb_with_struct_diplomatCallback_f_Native): Int
+    fun CallbackWrapper_test_multiple_cb_args(f: DiplomatCallback_CallbackWrapper_test_multiple_cb_args_diplomatCallback_f_Native, g: DiplomatCallback_CallbackWrapper_test_multiple_cb_args_diplomatCallback_g_Native): Int
+}
+
+internal class CallbackWrapperNative: Structure(), Structure.ByValue {
+    @JvmField
+    internal var cantBeEmpty: Byte = 0;
+  
+    // Define the fields of the struct
+    override fun getFieldOrder(): List<String> {
+        return listOf("cantBeEmpty")
+    }
+}
+
+
+internal interface Runner_DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomatCallback_f: Callback {
+    fun invoke(ignored: Pointer?, arg0: Int ): Int
+}
+
+class DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomatCallback_f_Native: Structure(), Structure.ByValue {
+    @JvmField
+    internal var dataa: Pointer = Pointer(0L);
+    @JvmField
+    internal var run_callback: Runner_DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomatCallback_f
+        = object :  Runner_DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomatCallback_f {
+                override fun invoke(ignored: Pointer?, arg0: Int ): Int {
+                    throw Exception("ERROR NOT IMPLEMENTED")
```

Comment:
question: why this? is this a default value?

(perhaps worth improving the exception text)

Reply:
Good idea! Yes, it is a default value -- I'll make this clear in the exception string.